components/buttons/Button.tsx

```diff
@@ -1,15 +1,31 @@
 import Link from 'next/link';
 import { twMerge } from 'tailwind-merge';
 
+export enum ButtonSizeType {
+  small = 'small',
+  default = 'default',
+}
+
+export enum ButtonType {
+  submit = 'submit',
+  reset = 'reset',
+  button = 'button',
+}
+
+export enum ButtonIconPositionType {
+  left = 'left',
+  right = 'right',
+}
+
```

Comment:
You can move these to a separate file in the `~types/components/buttons/...`

Reply:
Done